pkg/repometrics/vulnerabilities.go

```diff
@@ -17,11 +17,14 @@ type Scanner interface {
 func (m *Metrics) ScanVulnerabilities(ctx context.Context, s Scanner) error {
 	var err error
 
-	vulns, err := s.Scan(ctx)
+	vulnerabilities, err := s.Scan(ctx)
 	if err != nil {
 		return fmt.Errorf("scanning vulnerabilities: %w", err)
 	}
-	m.Vulnerabilities = vulns
+	// m.Vulnerabilities = vulnerabilities //we need to produce a cycloneDX file from scanning, //nolint:gofumpt
+	// and have that file being picked up in same way as externally generated setups using SBOM, Vulnerability files and VEX //nolint:gofumpt
```

Comment:
Not sure if these comments need to be here?

Reply:
Actually a TODO probably communicates better, as this was entered for the scanning to be changed going forward to use the same principles as the SBOM, Vulnerability and VEX files coming from elssewhere e.g. inventory and curated sources for risk assessment in connection with vulnerabilities